packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -47,6 +47,10 @@ class MultipartUploader {
 
   #onReject = (err) => (err?.cause === pausingUploadReason ? null : this.#onError(err))
 
+  maxMultipartParts = 10000
+
+  minPartSize = 5 * MB
```

Comment:
I thought public field makes more sense because:
- it can be referenced and changed in tests
- a config value that can be references in general.

Perhaps a premature optimisation